compiler/lcalc/to_ocaml.ml

```diff
@@ -342,7 +342,8 @@ let rec format_expr (ctx : decl_ctx) (fmt : Format.formatter) (e : 'm expr) :
     Format.fprintf fmt
       "@[<hov 2>%a@ @[<hov 2>{filename = \"%s\";@ start_line=%d;@ \
        start_column=%d;@ end_line=%d; end_column=%d;@ law_headings=%a}@]@ %a@]"
-      Print.operator op
+      (Print.operator ~debug:true)
```

Comment:
Why `debug:true` here?

Reply:
Ah this is correct but indeed a bit unclean ;
- `Print.operator` prints the full operator symbols when `debug=true` (_i.e._ `+$`, `+^`, etc.)
- and the "simplified" operators otherwise, which are a bit better for readability, but won't work for OCaml output.

but that doesn't match the name for the `debug` flag at all anymore ; I'll add a different function (even if it does the same under the hood)